package.json

```diff
@@ -7,7 +7,26 @@
     "monorepo",
     "sui"
   ],
-  "scripts": {},
+  "scripts": {
+    "commit": "sui-mono commit",
+    "release": "sui-mono release",
+    "release:check": "sui-mono check",
+    "commitmsg": "validate-commit-msg"
```

Comment:
Exactly. Husky adds "native" scripts to npm scripts correponding to git hooks
https://github.com/typicode/husky/blob/master/HOOKS.md